CHANGES.rst

```diff
@@ -61,6 +61,10 @@ astropy.table
 
 astropy.tests
 ^^^^^^^^^^^^^
+- The function ``quantity_allclose`` was moved to the ``units`` package with the new,
+  shorter name ``allclose``. It is deprecated in ``tests``. [#7252]
+
+- The function ``_unquantify_allclose_arguments`` was moved to the ``units`` package. It is deprecated in ``tests``. [#7252]
```

Comment:
This is a private function, so we should not advertise it in any way (i.e., just remove this from the changelog).